app/views/pages/additional_guidance.html.erb

```diff
@@ -22,18 +22,24 @@
 
       <%= f.govuk_submit (additional_guidance_form.additional_guidance_markdown.blank? ? t("guidance.preview_guidance") : t("guidance.update_preview")), secondary: true,  name: :preview_markdown %>
 
-      <%= govuk_details(summary_text: "<h2 class='govuk-!-font-size-19 govuk-!-margin-0 govuk-!-font-weight-regular'>#{t("guidance.formatting_help.heading")}</h2>".html_safe) do %>
-
-        <% %w[links second_level_headings third_level_headings bulleted_lists numbered_lists].each do |format_name| %>
-          <h3 class="govuk-heading-m"><%= t("guidance.formatting_help.#{format_name}.heading") %></h3>
-
-          <p><%= simple_format(t("guidance.formatting_help.#{format_name}.instructions")) %></p>
-
-          <%= govuk_inset_text do %>
-            <%= simple_format(t("guidance.formatting_help.#{format_name}.example")) %>
+      <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+          <h2 class='govuk-details__summary-text govuk-!-font-size-19 govuk-!-margin-0 govuk-!-font-weight-regular'>
+            <%= t("guidance.formatting_help.heading") %>
+          </h2>
+        </summary>
```

Comment:
We should really be using the details component where we can. Maybe we could offer a PR upstream to toggle which tag to use inside the summary.

Reply:
In this case the govuk-components gem is behaving the same way as the Design System nunjucks macro, so we might need to get both to change their behaviour.

I've added a note about what we're doing to the Design System discussion of the component here: https://github.com/alphagov/govuk-design-system-backlog/issues/44#issuecomment-1664275484